fastestimator/pipeline.py

```diff
@@ -181,81 +215,84 @@ def benchmark(self,
         Args:
             mode: The execution mode to benchmark. This can be 'train', 'eval' or 'test'.
             epoch: The epoch index to benchmark. Note that epoch indices are 1-indexed.
+            ds_id: The ds_id to benchmark. If None, all ds_ids will be used.
```

Comment:
not clear whether this is really the desired behavior. Maybe is reasonable though since we don't allow None as a key where there are multiple and they can use the mode to select what to benchmark...

Reply:
now it's clear